src/main/scala/io/kaizensolutions/virgil/codecs/CassandraTypeMapper.scala

```diff
@@ -23,7 +23,7 @@ trait CassandraTypeMapper[Scala] {
   def toCassandra(in: Scala, dataType: DataType): Cassandra
   def fromCassandra(in: Cassandra, dataType: DataType): Scala
 }
-object CassandraTypeMapper {
+object CassandraTypeMapper extends LowPriorityTypeMapperInstances {
```

Comment:
This was causing false positives for implicit resolutions for IntelliJ's highlighter so doing this made it happier